db/people.yml

```diff
@@ -2608,6 +3080,22 @@ vgharakhanyan:
       status: ms
   name: Vahe Gharakhanyan
   position: Graduate Student
+vnield:
+  active: false
+  aka: []
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 0
+      institution: fix.me
```

Comment:
Let's change the name to Victoria M. Nield and put institution to unknown and degree to PhD